### ISSUE TITLE
UI/UX Overhaul/Consolodation

### ISSUE THREAD

**Issue by RyanTheAllmighty** [ATLauncher/ATLauncher#759]
### The Problem

Currently a lot of the UI components use patterns and designs which are wildly different from each other. Often being bad UX patterns and bad for end users being able to use the application.

### The Solution

There's a bunch of things that we need to consolidate to make the UI similar between the different areas in the application.

For instance a lot of dialogs don't have a separator between the content and the bottom bar with the action buttons. Some do, but most don't, they should all be the same.

The same can be said for the button placement. Sometimes the primary action that progresses the user forward is on the left, and sometimes on the right. Main actions should be on the right, with failure actions such as cancelling or closing, should be on the left.

There's also a large duplication of components in the filesystem, having a base set of components would be nice to start to consolidate common UI patterns.

TLDR; refactor, reuse and recycle components, make UI and UX better

Going to use this issue as a dumping ground of issues from users around UI/UX concerns as well as my own observations as to the shit that needs cleaning up.

### Have you checked this issue doesn't already exist?

- [X] I am sure this issue doesn't exist already

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
